ninja_simple_jwt/auth/token_user.py

```diff
@@ -0,0 +1,29 @@
+from typing import Any, Optional
+
+
+class TokenUser:
+
+    def __init__(self, user_id: Any = None, email: str | None = None, roles: list[str] | None = None):
+        self.id = user_id
```

Comment:
I think you may want to either document that if using the token user backend queries have to be updated to like `MyModel.objects.filter(related_user=token_user.id)` or I think you could try adding a `__int__` method to `TokenUser` so that Django can naturally convert it to an integer during query generation.

```
    def __int__(self):
        return self.id
```

This, I think, should allow for you to do `MyModel.objects.filter(related_user=token_user)`, like normal. You will want to test that, though.

Reply:
Nice